vue3-ts-quasar/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "vue-cli-service serve --mode=development",
```

Comment:
isn't serve in `development` mode by default?
https://cli.vuejs.org/guide/mode-and-env.html#modes